sim/hunter/cata_items.go

```diff
@@ -0,0 +1,47 @@
+package hunter
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+)
+
+var ItemSetLightningChargedBattleGear = core.NewItemSet(core.ItemSet{
+	Name: "Lightning-Charged Battlegear",
+	Bonuses: map[int32]core.ApplyEffect{
+		2: func(agent core.Agent) {
+			hunter := agent.(HunterAgent).GetHunter()
+			hunter.RegisterAura(core.Aura{
+				Label:    "T11 2pc",
+				Duration: core.NeverExpires,
+				OnReset: func(aura *core.Aura, sim *core.Simulation) {
+					aura.Activate(sim)
+				},
+				OnGain: func(aura *core.Aura, sim *core.Simulation) {
+					hunter.SerpentSting.BonusCritRating += 0.05
+				},
+				OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+					hunter.SerpentSting.BonusCritRating -= 0.05
+				},
```

Comment:
same comments as the 4pc bonus

Reply:
also when working with `BonusCritRating` its prefered to do `N * core.CritRatingPerCritChange`